scripts/baseDemo.ts

```diff
@@ -0,0 +1,363 @@
+import { v4 } from 'uuid';
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const graphVariant = 'main';
+export const client = new ApolloClient({
+  cache: new InMemoryCache(),
+});
+
+const ME_CHECK = gql(`
+  query ApiCheck {
+    me {
+      ...on User {
+        id
+        memberships {
+          account {
+            id
+          }
+        }
+      }
+      ...on Service {
+        id
+        title
+      }
+    }
+  }
+`);
```

Comment:
```suggestion
const ME_CHECK = gql`
  query ApiCheck {
    me {
      ...on User {
        id
        memberships {
          account {
            id
          }
        }
      }
      ...on Service {
        id
        title
      }
    }
  }
`;
```

Minor nit. No need for parens.